### PR TITLE
chore: purge rewards and format

### PR DESCRIPTION
**Pull request by Keyrxng**
Relates to https://github.com/ubiquity/.github/issues/98
> ###### I guess it doesn't really but sort of

- purged `/rewards`
- brought over the used constants and abis
- `yarn format` fixes

### CODE REVIEW

**Comment by rndquu** [comment]
@Keyrxng The build workflow is failing :(

**Comment by Keyrxng** [comment]
`yarn knip` is passing but the workflow isn't I'm unsure how to approach that but the build should pass now that I've removed the lib causing the issue

**Review by rndquu — APPROVED** [review]

File: .cspell.json [hunk 1]
```diff
@@ -35,9 +35,11 @@
...
     "xmark",
-    "SUPABASE"
+    "SUPABASE",
+    "Knip"
   ],
   "dictionaries": ["typescript", "node", "software-terms", "html"],
   "import": ["@cspell/dict-typescript/cspell-ext.json", "@cspell/dict-node/cspell-ext.json", "@cspell/dict-software-terms"],
-  "ignoreRegExpList": ["[0-9a-fA-F]{6}"]
+  "ignoreRegExpList": ["[0-9a-fA-F]{6}"],
+  "ignoreWords": ["wfzpewmlyiozupulbuur"]
```
Comment (0x4007) [inline .cspell.json]: Use an inline ignore directive in the file using it instead 

```
// cSpell: ignore wfzp...
```

File: .cspell.json [hunk 2]
```diff
@@ -35,9 +35,11 @@
...
     "XDAI",
     "xmark",
-    "SUPABASE"
+    "SUPABASE",
+    "Knip"
   ],
   "dictionaries": ["typescript", "node", "software-terms", "html"],
   "import": ["@cspell/dict-typescript/cspell-ext.json", "@cspell/dict-node/cspell-ext.json", "@cspell/dict-software-terms"],
-  "ignoreRegExpList": ["[0-9a-fA-F]{6}"]
+  "ignoreRegExpList": ["[0-9a-fA-F]{6}"],
```
Comment (0x4007) [inline .cspell.json]: This is for colors? Where is the issue?

**Review by 0x4007 — CHANGES_REQUESTED** [review]
Code looks mostly fine just cspell related nits

File: .cspell.json (hunk 2 above)
Comment (Keyrxng) [reply .cspell.json]: It's just because I've added a comma it's registering a full line change but no dramas

**Review by 0x4007 — APPROVED** [review]

**Comment by gitcoindev** [comment]
Hi @Keyrxng , the Knip configuration was fixed https://github.com/ubiquity/onboard.ubq.fi/pull/19 , most likely you will encounter merge conflicts in this area. After the merge conflicts are resolved I will also review and the pull request can be merged.

**Comment by Keyrxng** [comment]
@gitcoindev ready when you are

**Comment by Keyrxng** [comment]
Is this ready to be merged?

**Comment by gitcoindev** [comment]
@Keyrxng thank you for pinging me again, yes!

**Comment by Keyrxng** [comment]
> thank you for pinging me again

I can never tell if it's rude of me to be pinging or if that should be the default lmao I get you folks are busy but cheers

**Comment by gitcoindev** [comment]
> > thank you for pinging me again
> 
> I can never tell if it's rude of me to be pinging or if that should be the default lmao I get you folks are busy but cheers

In my case: If I do not reply within 48hrs, do ping me -)

**Pull request merged**